projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -23,7 +23,20 @@ export class IgxMaskDirective implements OnInit, AfterViewChecked, ControlValueA
      * ```
      */
     @Input('igxMask')
-    public mask: string;
+    public get mask(): string {
+        return this._mask || this.defaultMask;
```

Comment:
do not like that getter is actually sets the mask if there is no initial value 😢